Assets/Scripts/Explosion.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Explosion : MonoBehaviour {
+
+	private SpriteRenderer sr;
+
+	void Start() {
+		sr = GetComponent<SpriteRenderer> ();
+	}
+
+	void Update () {
+		Color c = sr.color;
+		c.a -= 3 * Time.deltaTime;
+		sr.color = c;
+
+		if (sr.color.a <= 0) {
+			Destroy (gameObject);
+		}
+	}
+
+	public void SetSize(float size) {
+		transform.localScale *= size;
```

Comment:
The circle sprite is 4x4 pixels and its pixels per unit is set to 1 so it takes up 4 units in the world space. 
so the prefab has the scale set to 0.25 so its diameter is exactly 1 unit.
PS. When we have real art they're have much higher pixels per unit.